src/euphorie/client/browser/configure.zcml

```diff
@@ -571,6 +571,14 @@
       template="templates/login.pt"
       layer="euphorie.client.interfaces.IClientSkinLayer"
       />
+  <browser:page
+      name="register"
+      for="*"
+      permission="zope.Public"
+      class=".login.Login"
+      template="templates/login.pt"
+      layer="euphorie.client.interfaces.IClientSkinLayer"
+      />
```

Comment:
Hi @thet, why is it necessary to have this definition?
It does not seem correct to me. If we don't need this for normal usage, why do we need it in the tests?

Reply:
I see now, it was how you fixed client: test_functional_login.RegisterTests
I will try something different: Since the whole login / register is all one view now, I will try to just call the login view instead.